store/index-refactor.js

```diff
@@ -35,7 +36,12 @@ export const getters = {
             return "";
         }
         return description;
+    },
+
+    getColumnNames(p_state) {
+        return (0 === p_state.dataTable.length) ? [] : Object.keys(p_state.dataTable[0]);
```

Comment:
I mentioned the same thing in #291: is there any reason to expect that this `getter` is called before `dataTable` is set to something non-empty? If yes, we should decide whether that's something that should work or not break. And then we should test that our components are still happy if they get an empty array back. 

My sense is that we don't need to worry about this unless there is a good reason this getter could be called without `dataTable` being set. But let me know if you disagree.

Reply:
Answered by Jonathan in #291